src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -84,6 +87,16 @@ public AdyenConfigProperties(final Properties properties) {
         this.hppTarget = properties.getProperty(PROPERTY_PREFIX + "hpp.target");
         this.hppSkipDetailsTarget = this.hppTarget != null ? this.hppTarget.replace(this.hppTarget.substring(this.hppTarget.lastIndexOf('/') + 1), "skipDetails.shtml") : null;
         this.hppVariantOverride = properties.getProperty(PROPERTY_PREFIX + "hppVariantOverride");
+
+        int hppExpirationPeriodInDays = 0;
+        try {
+            final String hppExpirationPeriod = properties.getProperty(PROPERTY_PREFIX + "hppExpirationPeriodInDays");
+            hppExpirationPeriodInDays = Integer.parseInt(hppExpirationPeriod);
+        } catch(NumberFormatException e) {
+            hppExpirationPeriodInDays = DEFAULT_HPP_EXPIRATION_PERIOD_IN_DAYS;
+        }
+        this.hppExpirationPeriodInDays = hppExpirationPeriodInDays;
```

Comment:
Yeah, per payment method configuration should be possible, we store the a `paymentMethod` attribute in the `additional_data` field (same for `countryCode`).

I guess for now, we could leave it like this. It should be easy to update this without breaking backwards compatibility. What do you think?

In the meantime, I'll update the default value.